jaeger-micrometer/src/test/java/com/uber/jaeger/micrometer/MicrometerTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2016, Uber Technologies, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.uber.jaeger.micrometer;
+
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import com.uber.jaeger.metrics.Metrics;
+import io.micrometer.prometheus.PrometheusConfig;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import org.hamcrest.core.IsEqual;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MicrometerTest {
+  PrometheusMeterRegistry prometheusRegistry;
+  MicrometerStatsReporter metricsReporter;
+  Metrics metrics;
+
+  @Before
+  public void setUp() {
+    prometheusRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
+    io.micrometer.core.instrument.Metrics.addRegistry(prometheusRegistry);
+    metricsReporter = new MicrometerStatsReporter();
+    metrics = Metrics.fromStatsReporter(metricsReporter);
+  }
+
+  @After
+  public void tearDown() {
+    io.micrometer.core.instrument.Metrics.removeRegistry(prometheusRegistry);
+  }
+
+  @Test
+  public void testCounterWithoutExplicitTags() {
+    metrics.decodingErrors.inc(1);
+    assertThat(metricsReporter.registry()
+          .get("jaeger:span_context_decoding_errors").counter().count(),
+          IsEqual.equalTo(1D)
```

Comment:
Curious why incrementing the decoding errors by 1 translates to 10 in the counter? Same applies in following tests.
If there is a good reason, might be worth documenting it in the test, as doesn't look right.